lib/js/src/protocol/SdlProtocolBase.js

```diff
@@ -412,6 +415,18 @@ class SdlProtocolBase {
      */
     _handleStartServiceNAK (sdlPacket) {
         const serviceType = sdlPacket.getServiceType();
+        const protocolVersion = sdlPacket.getVersion();
+        if (protocolVersion >= 5) {
+            let rejectedTag = null;
+            let rejectedParams;
+            if (serviceType === ServiceType.AUDIO) {
+                rejectedTag = ControlFrameTags.Audio.StartServiceNAK.REJECTED_PARAMS;
+            }
+            if (rejectedTag) {
+                rejectedParams = sdlPacket.getTag(rejectedTag);
+                console.error('Got StartSessionNAK with rejected params', rejectedParams);
+            }
+        }
```

Comment:
Appears that this logic is supposed to be part of the `_handleEndServiceNAK` method rather than the `_handleStartServiceNAK` method. See Java Suite's `handleServiceEndedNAK` method in `SdlProtocolBase.java`.